lib/src/config/misc.toml

```diff
@@ -12,6 +12,8 @@ register_snapshot_trigger = false
 [git]
 abandon-unreachable-commits = true
 auto-local-bookmark = false
+subprocess = false
+executable_path = "git"
```

Comment:
nit: `executable-path` (use dash instead of underscore in config keys)

Reply:
(only a little related): the option under core-watchman has underscores also...